python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bgp.schema.yml

```diff
@@ -1439,6 +1439,444 @@ keys:
                   description: |
                     Routes learned by the OSPF protocol.
                     The `ospf_route_type` is valid for source_protocols 'ospf' and 'ospfv3'.
+      address_family_ipv4_labeled_unicast:
+        type: dict
+        keys:
+          aigp_session:
+            type: dict
+            keys:
+              confederation:
+                type: bool
+              ebgp:
+                type: bool
+              ibgp:
+                type: bool
+          bgp:
+            type: dict
+            keys:
+              additional_paths:
+                type: dict
+                description: BGP additional-paths commands.
+                keys:
+                  receive:
+                    type: bool
+                    description: Receive multiple paths.
+                  send:
+                    type: dict
+                    description: Send multiple paths.
+                    keys:
+                      any:
+                        type: bool
+                        description: Any eligible path.
+                      backup:
+                        type: bool
+                        description: Best path and installed backup path.
+                      ecmp:
+                        type: bool
+                        description: All paths in best path ECMP group.
+                      ecmp_limit:
+                        type: int
+                        description: Amount of ECMP paths to send.
+                        convert_types:
+                          - str
+                        min: 2
+                        max: 64
+                      limit:
+                        type: int
+                        description: Amount of paths to send.
+                        convert_types:
+                          - str
+                        min: 2
+                        max: 64
```

Comment:
same for the block below

Reply:
@laxmikantchintakindi can you help update this PR to follow the same model as you are implementing for other areas in #3730? This will save us from having to deprecate these later, but just doing the new model directly. Thanks.

Reply:
Added the changes according to the new model. Added the defs_bgp_additional schema in this PR.